frontend/src/api/calls/studentCalls.ts

```diff
@@ -9,6 +13,12 @@ export interface IStudentQueryParams {
     unmatched: boolean;
 }
 
+export type IAllStudentInfo = {
```

Comment:
```suggestion
export type ICompleteStudentInfo = {
```

Reply:
I'll change it to IFullStudentInfo to stay consitent with the project version